ui/src/shared/utils/pageTitles.ts

```diff
@@ -3,7 +3,7 @@ import {store} from 'src/index'
 
 export const pageTitleSuffixer = (pageTitles: string[]): string => {
   const state = store.getState()
-  const currentOrg = get(state, 'orgs.org.name', '')
+  const currentOrg = get(state, 'resources.orgs.org.name', null)
   const titles = [...pageTitles, currentOrg, 'InfluxDB 2.0']
```

Comment:
use the `getOrg` selector please  : )

Reply:
https://github.com/influxdata/influxdb/blob/master/ui/src/organizations/selectors/index.ts#L7